src/couch_server.erl

```diff
@@ -271,20 +268,18 @@ all_databases(Fun, Acc0) ->
     {ok, FinalAcc}.
 
 
-make_room(Server, Options) ->
-    case lists:member(sys_db, Options) of
-        false -> maybe_close_lru_db(Server);
-        true -> {ok, Server}
-    end.
+make_room(Server, _Options) ->
```

Comment:
Since `make_room/2` is only called by couch_server, can't we safely remove this?